comps/cores/proto/docarray.py

```diff
@@ -14,6 +14,21 @@ class TextDoc(BaseDoc):
     text: str
 
 
+class MultiTextDoc(BaseDoc):
```

Comment:
I'm not sure that we should add these types to docarray package. Since these are very specific to PredictionGuard, I think it would be better to put them in comps/guardrails/factuality/predictionguard/factuality_predictionguard.py

Reply:
Fixed.

Reply:
What about PIIDoc? That's also fairly specific to PG.